include/libsigrok/libsigrok.h

```diff
@@ -836,7 +836,11 @@ enum sr_configkey {
 	/** Trigger matches. */
 	SR_CONF_TRIGGER_MATCH,
 
-	/** The device supports setting its sample interval, in ms. */
+	/**
+	 * The device supports setting its sample interval, in ms.
+	 * This is only used for devices that only support <1Hz sample rates.
+	 * @sa SR_CONF_SAMPLERATE
```

Comment:
Could you also add a cross reference from SR_CONF_SAMPLERATE  to this config key?

Reply:
not really?  I don't even have this code locally. consider this patch public domain, I wsa just trying to help it along.